app/objects/c_agent.py

```diff
@@ -97,11 +120,9 @@ def store(self, ram):
         self.update('executors', kwargs.get('executors'))
 
     async def gui_modification(self, **kwargs):
-        self.update('group', kwargs.get('group'))
-        self.update('trusted', kwargs.get('trusted'))
-        self.update('sleep_min', int(kwargs.get('sleep_min')))
-        self.update('sleep_max', int(kwargs.get('sleep_max')))
-        self.update('watchdog', int(kwargs.get('watchdog')))
+        loaded = self.AgentSchema(only=('group', 'trusted', 'sleep_min', 'sleep_max', 'watchdog')).load(kwargs)
+        for k, v in loaded.items():
+            self.update(k, v)
```

Comment:
yea this makes things easier

Reply:
so no further change to this line, then?